backend/routes/api.php

```diff
@@ -17,3 +18,5 @@
 Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+Route::apiResource('users', UserController::class)->only(['index']);
```

Comment:
I think users API should not be public and only be accessed by the admin based on our features.